README

```diff
@@ -245,7 +275,12 @@
          - The user adds a line "avocado-misc-tests/generic/ras.py" (without the quotes) to the end of the config/tests/host/sanity.cfg 
          file in this tests repository and saves the file.
          - The user runs the host_sanity suite using the avocado-setup.py script explained in Section 4 above.
-
+      
+      OPTION #2: If you want to run your own unique list of host tests without changing the config/tests/host/sanity.cfg file,
+      create your own config file based on the structure & syntax of the sanity.cfg file and then reference that specific file 
+      when you run avocado-setup.py using the "--input-file" option explained above.
```

Comment:
Usage of input file needs to explained bit @narasimhan-v, could you pls help on thi

Reply:
@sathnaga Sure.
@rquander As stated above, this is misguiding and is a sensitive section. I can provide a detailed explanation and example.